bindings/python/src/OpenSpaceToolkitPhysicsPy/Environment/Atmospheric/Earth/Manager.cpp

```diff
@@ -0,0 +1,53 @@
+/// Apache License 2.0
+
+#include <OpenSpaceToolkit/Physics/Environment/Atmospheric/Earth/Manager.hpp>
+
+inline void OpenSpaceToolkitPhysicsPy_Environment_Atmospheric_Earth_Manager(pybind11::module& aModule)
+{
+    using namespace pybind11;
+
+    using ostk::core::types::Shared;
+
+    using ostk::physics::environment::atmospheric::earth::Manager;
+
+    class_<Manager> manager(aModule, "Manager");
+
+    manager
+
+        .def("get_mode", &Manager::getMode)
+        .def("get_local_repository", &Manager::getLocalRepository)
+        .def("get_cssi_space_weather_directory", &Manager::getCSSISpaceWeatherDirectory)
+        .def("get_remote_url", &Manager::getRemoteUrl)
+        .def("get_cssi_space_weather_array", &Manager::getCSSISpaceWeatherArray)  // TBI: check case
```

Comment:
What is this TBI?

Reply:
Oh gotcha yeah. Thanks that makes sense I agree with lower case CSSI.

Reply:
good catch